src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/__init__.py

```diff
@@ -127,10 +127,13 @@ def register_common_storage_account_options(self):
 
 
 class StorageCommandGroup(AzCommandGroup):
-    def storage_command(self, name, method_name=None, command_type=None, oauth=False, **kwargs):
+    def storage_command(self, name, method_name=None, command_type=None, oauth=False, generic_update=None, **kwargs):
         """ Registers an Azure CLI Storage Data Plane command. These commands always include the four parameters which
         can be used to obtain a storage client: account-name, account-key, connection-string, and sas-token. """
-        if command_type:
+        if generic_update:
+            command_name = '{} {}'.format(self.group_name, name) if self.group_name else name
```

Comment:
Why is this necessary for the `if generic_update:` branch but not the `else:` branch?

Reply:
Just really the different types of commands we currently have in our CLI, reg commands, custome_commands, generic_update (also wait commands, and show commands)